crates/ghost_actor/src/ghost_tracker.rs

```diff
@@ -28,22 +28,50 @@ struct GhostTrackerEntry<Context, CbData, E> {
     cb: GhostCallback<Context, CbData, E>,
 }
 
-/// GhostTracker registers callbacks associated with request_ids
-/// that can be triggered later when a response comes back indicating that id
-pub struct GhostTracker<Context, CbData, E> {
+#[derive(Debug, Clone)]
+pub struct GhostTrackerBuilder {
     request_id_prefix: String,
-    pending: HashMap<RequestId, GhostTrackerEntry<Context, CbData, E>>,
+    default_timeout: std::time::Duration,
 }
 
-impl<Context, CbData, E> GhostTracker<Context, CbData, E> {
-    /// create a new tracker instance (with request_id prefix)
-    pub fn new(request_id_prefix: &str) -> Self {
+impl Default for GhostTrackerBuilder {
+    fn default() -> Self {
         Self {
-            request_id_prefix: request_id_prefix.to_string(),
+            request_id_prefix: "".to_string(),
+            default_timeout: std::time::Duration::from_millis(2000),
```

Comment:
maybe make a constant for this timeout value, just to make it slightly easier to dig up when debugging timeout conditions. Although perhaps the `Default` trait encapsulates it enough already.

Reply:
:+1: